misaligned/misaligned.java

```diff
@@ -1,18 +1,21 @@
-public class Misaligned {
-    static int printColorMap() {
-        String majorColors[] = {"White", "Red", "Black", "Yellow", "Violet"};
-        String minorColors[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-        int i = 0, j = 0;
-        for(i = 0; i < 5; i++) {
-            for(j = 0; j < 5; j++) {
-                System.out.printf("%d | %s | %s\n", i * 5 + j, majorColors[i], minorColors[i]);
-            }
-        }
-        return i * j;
-    }
-    public static void main(String[] args) { 
-        int result = printColorMap();
-        assert(result == 25);
-        System.out.println("All is well (maybe!)");
-    }
-}
+public class misaligned {
+	static int printColorMap() {
+		String majorColors[] = { "White", "Red", "Black", "Yellow", "Violet" };
+		String minorColors[] = { "Blue", "Orange", "Green", "Brown", "Slate" };
+		int i = 0, j = 0;
+		for (i = 0; i < 5; i++) {
+			for (j = 0; j < 5; j++) {
+				String pair = (i * 5 + j) + "|" + majorColors[i] + "|" + minorColors[i];
+				MisalignedTest.checkPair(pair, majorColors, minorColors);

```

Comment:
Test code is invoked from within production code i.e. production code has clear dependency on test code.
This dependency should not be present.
Maybe you can cache the color map and verify at the end. Please think over ...

Reply:
Resolved, Could you please validate now. Thankyou!!